### PR TITLE
Add event subscribe page

### LINKED ISSUE #1834: New customer meeting landing page
We need a simple landing page where customers can signup for three possible 'rondetafelbijeenkomsten', the first one taking place early June. I would suggest to use the endpoint **rondetafelgesprek**

- [ ] Add a form:
Fields:
- Date (dropdown) 
- Name (text)
- Function title (text)
- Organisation (text)
- Email address (text)
- Notes (comment; for possible allergies)) 

- [ ] Add completion message thanking for their signup. (content @michellusthof & @aknoeff)

@RonaldDijkstra: would it be possible to have 1 person signup a colleague using the same form?

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Register the page 
- [x] Add hero
- [x] Add form 
  - [x] Add possibility to register multiple people 
  - [x] Add unique value per added co-worker so formkeep actually can capture the name value
  - [x] Remove extra field options
  - [x] Custom thanks message 
- [x] Real path 
- [x] Real content
- [x] Block search engines (invite only event, so no other registrars) 

Close #1834

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
As a side project we introduce a hero variant style in this PR: `variant-normal`. 
This can be used in other pages as well if we introduce the partial for hero at more pages.

**Comment by RonaldDijkstra** [comment]
Small mistake on thanks page

**Comment by RonaldDijkstra** [comment]
Fixed

**Comment by RonaldDijkstra** [comment]
Deploy preview: https://deploy-preview-1861--defacto-nl.netlify.com/informatiesessie

**Comment by michellusthof** [comment]
We need to add one additional field to the form: job title (functie). And on mobile, the hero text is shown partially truncated.

**Comment by michellusthof** [comment]
This shouldn't delay reviewing the new structure and logic introduced by Ronald.

**Comment by RonaldDijkstra** [comment]
That personeelsontwikkeling/othertext is crazy wide

**Comment by RonaldDijkstra** [comment]
> We need to add one additional field to the form: job title (functie). 

Why do we **need** to add this?

**Comment by RonaldDijkstra** [comment]
@michellusthof position is added. You can change the headline.

File: locales/nl.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ nl:
     job_apply: https://formkeep.com/f/da32eaeda39e
     capp_agile: https://formkeep.com/f/0f2125be5045
     upgrade_productsheet: https://formkeep.com/f/ed6e8e1efeeb
+    information_session: https://formkeep.com/f/bc26f14079fd
```
Comment (RonaldDijkstra) [inline locales/nl.yml]: This ref is not used.

File: source/javascripts/_forms.js.erb [hunk 2]
```diff
@@ -14,13 +14,34 @@ $(function () {
   var $forms = $('form');
 
   // Change option color on select
-  var demoSelect = document.getElementById('select');
-  if (demoSelect) {
-    demoSelect.onchange = function () {
-      demoSelect.className = this.options[this.selectedIndex].className;
+  var select = document.getElementById('select');
```
Comment (RonaldDijkstra) [inline source/javascripts/_forms.js.erb]: Changed demoSelect (which was not in use anymore) to select to target all elements with id="select"

File: source/javascripts/_forms.js.erb [hunk 3]
```diff
@@ -14,13 +14,34 @@ $(function () {
...
+  if (select) {
+    select.onchange = function () {
+      select.className = this.options[this.selectedIndex].className;
     }
   }
 
+  // Add co-worker fields to subscribe form
+  $("#add-button").click(function(){
+    $(".extra-person-placeholder").append(
+      "<h3>Voeg een collega toe</h3>" +
```
Comment (RonaldDijkstra) [inline source/javascripts/_forms.js.erb]: Should be from translation.

File: source/localizable/bedankt.html.erb [hunk 4]
```diff
@@ -52,6 +52,15 @@ document.addEventListener('DOMContentLoaded', function() {
     // Replace message with newsletter message if it was a newsletter action
     messageEl.innerHTML = message || '';
   }
+  <% if root_locale == :nl %>
```
Comment (RonaldDijkstra) [inline source/localizable/bedankt.html.erb]: This is only active in dutch lang for now

File: source/partials/_hero.erb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<div class="hero <%= variant %>" role="banner" style="background-image: url(<%= image_path("moods/" + locals[:background_image] || "moods/defacto.jpg") %>)">
```
Comment (RonaldDijkstra) [inline source/partials/_hero.erb]: The new hero partial!

File: source/stylesheets/base/_form-elements.scss [hunk 6]
```diff
@@ -175,3 +175,26 @@ input.invalid {
 .placeholder {
   color: $placeholder;
 }
+
+select {
```
Comment (RonaldDijkstra) [inline source/stylesheets/base/_form-elements.scss]: Moved from _forms as this is an element

**Comment by michellusthof** [comment]
> > We need to add one additional field to the form: job title (functie).
> 
> Why do we **need** to add this?

To ensure the right people will sign-up for the meeting. We only want decision makers, no key users.

**Comment by RonaldDijkstra** [comment]
> To ensure the right people will sign-up for the meeting. We only want decision makers, no key users.

The extra field does not ensure that.

**Pull request merged**